src/test/java/ddingdong/ddingdongBE/domain/feed/repository/FeedRepositoryTest.java

```diff
@@ -60,4 +60,58 @@ void findAllByClubIdOrderById() {
     Assertions.assertThat(feeds.get(2).getThumbnailUrl()).isEqualTo("썸네일1");
     Assertions.assertThat(feeds.get(2).getId()).isEqualTo(1L);
   }
+
+  @DisplayName("모든 동아리의 최신 피드를 모두 조회할 수 있다.")
+  @Test
+  void test() {
+    // given
+    Club club1 = fixture.giveMeBuilder(Club.class)
+        .set("name", "카우1")
+        .set("user", null)
+        .sample();
+    Club club2 = fixture.giveMeBuilder(Club.class)
+        .set("name", "카우2")
+        .set("user", null)
+        .sample();
+    Club club3 = fixture.giveMeBuilder(Club.class)
+        .set("name", "카우3")
+        .set("user", null)
+        .sample();
+    Club savedClub1 = clubRepository.save(club1);
+    Club savedClub2 = clubRepository.save(club2);
+    Club savedClub3 = clubRepository.save(club3);
+
+    Feed feed1 = fixture.giveMeBuilder(Feed.class)
+        .set("club", savedClub1)
+        .set("thumbnailUrl", "클럽 1 올드 url")
+        .sample();
+    Feed feed2 = fixture.giveMeBuilder(Feed.class)
+        .set("club", savedClub1)
+        .set("thumbnailUrl", "클럽 1 최신 url")
+        .sample();
+    Feed feed3 = fixture.giveMeBuilder(Feed.class)
+        .set("club", savedClub2)
+        .set("thumbnailUrl", "클럽 2 올드 url")
+        .sample();
+    Feed feed4 = fixture.giveMeBuilder(Feed.class)
+        .set("club", savedClub2)
+        .set("thumbnailUrl", "클럽 2 최신 url")
+        .sample();
+    Feed feed5 = fixture.giveMeBuilder(Feed.class)
+        .set("club", savedClub3)
+        .set("thumbnailUrl", "클럽 3 올드 url")
+        .sample();
+    Feed feed6 = fixture.giveMeBuilder(Feed.class)
+        .set("club", savedClub3)
+        .set("thumbnailUrl", "클럽 3 최신 url")
+        .sample();
+    feedRepository.saveAll(List.of(feed1, feed2, feed3, feed4, feed5, feed6));
+    // when
+    List<Feed> newestFeeds = feedRepository.findNewestAll();
+    // then
+    Assertions.assertThat(newestFeeds.size()).isEqualTo(3);
+    Assertions.assertThat(newestFeeds.get(0).getId()).isEqualTo(6L);
+    Assertions.assertThat(newestFeeds.get(1).getId()).isEqualTo(4L);
+    Assertions.assertThat(newestFeeds.get(2).getId()).isEqualTo(2L);
+  }
 }
```

Comment:
p4) EOF 컨벤션이 체크가 안되는 것 같은데 린트 설정에서 맨 마지막 라인에 한 줄 자동으로 추가하는 옵션이 있을 거에요. 그거 세팅 해두시면 좋을 것 같습니다!!
```suggestion
}

```

Reply:
죄송합니다. 확인한다고 하였는데, 잘 안됐네요. 설정 추가해보겠습니다